### ISSUE TITLE
[BUG] Brief description of your issue

### ISSUE THREAD

**Issue by stevethibault** [ironmonk88/monks-enhanced-journal#350]
**Describe the bug**
If I create a journal entry, of any kind, and drag it on to a scene, the icon will dissapear after certian actions. 
When I refresh foundry, when I log out and log in, sometimes when I change scenes, and return back.

**To Reproduce**
1. Create a journal entry of encounter, or enhanced journal, possibly others, only tested these two.
2. Create a foundry vtt scene with an image
3. Drag the created entry onto the scene so that an icon is created
4. Click on "Update Map Note" in the popup
5. Hit the browser refresh button, or log out and log back into foundry, or switch to a different scene and back again(intermittent)
6. The journal icon will not be visible on the scene
7. Drag another journal entry onto the scene, and either close the popup, or click on "Update Map Note"
8. The original journal item icon will now reappear, and if Update was clicked the new one will also.

**Expected behavior**
The icon should be visible and in the location it originally was after the action taken in step 5
**Screenshots**

**Please complete as much of the following information as possible:**
 - De googled Chromium v 101.0.4951.64
 - Foundry Version v9.269
 - System D&D5e 1.6.1
 - Module vesion Monks Enhanced Journal 1.0.65
 - Are there any errors in the console, no.
 - I disabled all modules except for the monks suite of modules.

**Comment by ironmonk88** [comment]
I can't replicate.
Do you have Toggle Notes Display turned on?
![image](https://user-images.githubusercontent.com/75920956/209772217-2fe813ee-addf-4771-91fc-8d6008d0f558.png)
If it's turned off, reloading the page will switch to the Token layer, and notes won't be visible on that layer unless the setting is turned on.
And dragging a new Note onto the canvas will switch to the Notes layer, which will reveal the old Note.

**Comment by stevethibault** [comment]
Yep, that’s it. Sorry I missed this. I’m dumb. On Dec 28, 2022, at 2:05 AM, ironmonk88 ***@***.***> wrote:﻿
I can't replicate.
Do you have Toggle Notes Display turned on?

If it's turned off, reloading the page will switch to the Token layer, and notes won't be visible on that layer unless the setting is turned on.
And dragging a new Note onto the canvas will switch to the Notes layer, which will reveal the old Note.
